python/desc/imsim/imSim.py

```diff
@@ -6,7 +6,10 @@
 import sys
 import warnings
 from collections import namedtuple, defaultdict
-import pickle
+try:
+    import cPickle as pickle
+except:
+    import pickle
```

Comment:
FYI, I recommended Josh switch to this pattern.  AFAIU, cPickle tends to be equivalent functionality, but much faster.  So if it is available, it is better to use it.  (And it's basically always available unless you're using pypy I think.)
cf. https://pymotw.com/2/pickle/